<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKxSearch</class>
 <widget class="QWidget" name="QKxSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLineEdit" name="key"/>
   </item>
   <item>
    <widget class="QCheckBox" name="ascase">
     <property name="text">
      <string>Case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="regular">
     <property name="text">
      <string>Regular</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="findPrev">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qkxterm.qrc">
       <normaloff>:/kxterm/image/prev.png</normaloff>:/kxterm/image/prev.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="findNext">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qkxterm.qrc">
       <normaloff>:/kxterm/image/next.png</normaloff>:/kxterm/image/next.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="findAll">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qkxterm.qrc">
       <normaloff>:/kxterm/image/find.png</normaloff>:/kxterm/image/find.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="close">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qkxterm.qrc">
       <normaloff>:/kxterm/image/close.png</normaloff>:/kxterm/image/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qkxterm.qrc"/>
 </resources>
 <connections/>
</ui>
